src/common/dom/apply_themes_on_element.ts

```diff
@@ -51,31 +59,19 @@ export const applyThemesOnElement = (
         return;
       }
       const prefixedRgbKey = `--${rgbKey}`;
```

Comment:
Can we do this theme processing just once per theme (and not when we load them all)? Seems expensive to do it each time we apply a theme.

(I know, it feels like I never took a look at this code before :))